lib/Doctrine/ORM/Query/ParserResult.php

```diff
@@ -65,8 +65,8 @@ public function __construct()
     /**
      * Gets the ResultSetMapping for the parsed query.
      *
-     * @return ResultSetMapping|null The result set mapping of the parsed query or NULL
-     *                               if the query is not a SELECT query.
+     * @return ResultSetMapping The result set mapping of the parsed query or
+     *                          NULL if the query is not a SELECT query.
```

Comment:
Comment and change don't make sense together

Reply:
:thinking: indeed. The property is private, and from reading its accessor and the setter, it does not look like it could be null, so let's remove that part of the comment.